src/docs/product/integrations/source-code-mgmt/github/index.mdx

```diff
@@ -231,6 +231,22 @@ This feature will be automatically enabled for you once your GitHub integration
 
 ![How to disable Sentry comments on GitHub pull requests](github-org-settings.png)
 
+### Missing Member Detection
+
+If your organization has users committing to Github repositories linked to Sentry that are not members of your organization, Sentry will remind organization owners and managers to invite these users through a banner on the members list page.
+
+![Github missing members invite banner](github-missing-members-banner.png)
+
+The banner will show the 5 most active contributors. To see all of the missing members (up to 50), click **View All** on the banner. To snooze the banner, click the ellipsis > **Hide Missing Members**.
```

Comment:
```suggestion
The banner shows the 5 most active contributors that are not yet in the organization. To see all missing members (up to 50), click "View All" in the banner. To snooze the banner, click the ellipsis, then select "Hide Missing Members".
```

Reply:
So usually you should bold UI elements, but for some reason our docs only bold page names and we put UI elements in quotations lol. Suggested these edits for consistency.